src/components/map/MapService.js

```diff
@@ -843,13 +854,12 @@ goog.require('ga_urlutils_service');
          * layers.
          */
         this.getBackgroundLayers = function() {
-          var self = this;
-          if (!layers) {
-            return;
-          }
+          var that = this;
           return $.map(gaTopic.get().backgroundLayers, function(bodId) {
-            var retVal = {id: bodId,
-                          label: self.getLayerProperty(bodId, 'label')};
+            var retVal = {
+              id: bodId,
+              label: that.getLayerProperty(bodId, 'label')
+            };
```

Comment:
I thnik things have changed since we have the new bg layer selector. We don't care (don't know if this is wanted) of the list of bg layers of the topic, we only care of the first element to determine which one is displayed by default.

The list of bg layers are currently always the same (hardcoded) and so the translation id are defined in this list.